src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/ajax/AjaxRemoveItemSuccessResponse.java

```diff
@@ -0,0 +1,22 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ajax;
+
+/**
+ * UI Response for successfully removing an item. Will return the internationalized
+ * success text.
+ */
+public class AjaxRemoveItemSuccessResponse extends AjaxResponse {
```

Comment:
Why are you using a custom class for this, it looks pretty much identical to `AjaxSuccessResponse`

Reply:
Which extends to `AjaxUpdateItemSuccessResponse`, they are all the same.

Reply:
Updated in 4e0f1cd19cfb95ba3af35a86f7e3a2271d907412